.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
 
 # Build a production build of the project
 before_script:
-  - ng build --prod
```

Comment:
Please add ```--aot```.

Reply:
Actually there is no need of adding `--aot` I have gone through it and found that if we use production build then `--aot` is already used, so there is no need of specifying it. Have a look here https://angular.io/guide/aot-compiler

Reply:
I have also found that there is no difference in compiled files by using `ng build --prod` and `ng build --prod --aot`